packages/font-glyphs/src/letter/latin/x.ptl

```diff
@@ -63,6 +63,9 @@ glyph-block Letter-Latin-X : begin
 
 		local fine : [AdviceStroke 3] * (sw / Stroke)
 
+		local leftX  : _leftX  - [if fSlab SideJut 0]
```

Comment:
I think we should only extend the chancey strokes to "cover" serifs only if there's serif. So for cv58#13 we should use the original width.

And maybe when we do extension, we should not modify the position of the curved turnings. We should perhaps create a new variant of `leftX`/`rightX` value that adds `SideJut`, and modify the flat ends' code into something like

```ptl
     ....
     flat (rightX - TINY) (rightY + sign * sw / 2) [widths.center sw]
     curl rightXExt (rightY + sign * sw / 2) [widths.center sw]
```